src/frontend/components/VideoCard/VideoCard.js

```diff
@@ -28,7 +70,8 @@ export const VideoCard = ({ id, title, creator }) => {
 
                 <li>
                     <span className='material-icons'>watch_later</span>
-                    <span >Watch Later</span>
+                    {!inWatchlater && <span onClick={() => auth.isAuthenticated ? addToWatcherLater(video) : navigate("/login")} >Add to Watch later</span>}
+                    {inWatchlater && <span onClick={() => auth.isAuthenticated ? deleteFromWatchlater(video) : navigate("/login")}>Delete From Watch Later</span>}
```

Comment:
use prettier for code formatting, this is not readable

Reply:
@manish591 removed this long inline callbacks and added functions for better readability. Have a look